components/leaderboard/LeaderboardRow.tsx

```diff
@@ -1,34 +1,177 @@
+        color={avatarColor}
+        classes={[
+          'max-w-[3.125rem]',
+          'flex',
+          'bg-transparent',
+          'border',
+          'border-black',
+          'min-w-[3.125rem]',
+          'm-2',
+          'flex-none',
+          'md:hidden',
+        ]}
+      />
+      <div className={clsx('flex', 'justify-between', 'w-full', 'md:hidden')}>
+        <LabeledText label="RANK">{rankStr}</LabeledText>
+        <LabeledText label="TOTAL POINTS" childClassName="text-right">
+          {pointsStr}
+        </LabeledText>
+      </div>
+      {/* elements for non-mobile */}
+      <div
+        className={clsx(
+          'hidden',
+          'md:inline',
+          'font-extended',
+          'text-2xl',
+          'w-16',
+          'sm:w-24'
+        )}
+      >
+        {rankStr}
+      </div>
+
+      <div
+        className={clsx(
+          'flex',
+          'flex-1',
+          'items-center',
+          'font-extended',
+          'text-2xl',
+          'overflow-hidden'
+        )}
+      >
+        <div className={clsx('py-3', 'mr-5', 'hidden', 'md:inline')}>
+          <RawFishAvatar
+            color={avatarColor}
+            classes={[
+              'max-w-[3.125rem]',
+              'flex',
+              'bg-transparent',
+              'border',
+              'border-black',
+              'md:min-w-[3.125rem]',
+              'md:max-w-[3.125rem]',
+              'md:max-h-[3.125rem]',
+            ]}
+          />
+        </div>
+        <div className={clsx('hidden', 'md:inline', 'truncate')}>
+          {graffiti}
+        </div>
+      </div>
+      <div
+        className={clsx(
+          'hidden',
+          'md:inline',
+          'font-extended',
+          'text-2xl',
+          'ml-2'
+        )}
+      >
+        {pointsStr}
+      </div>
+    </div>
+    /*
+    <div
+      className={clsx("relative", "bg-white", "rounded", "flex", "items-center", "px-4", "sm:px-10")}
       style={{ boxShadow: '0 4px 4px 0 rgba(0, 0, 0, 0.19)' }}
     >
-      <div className="absolute inset-0 rounded border hover:border-2 border-black" />
-      <div className="font-extended text-2xl w-16 sm:w-24">{rankStr}</div>
-      <div className="flex flex-1 items-center font-extended text-2xl overflow-hidden">
-        <div className="py-3 mr-5">
-          <FishAvatar color={avatarColor} />
+      <div className={clsx("absolute", "inset-0", "rounded", "border", "hover:border-2", "border-black")} />
+      <div className={clsx("font-extended", "text-2xl", "w-16", "sm:w-24")}>{rankStr}</div>
+      <div className={clsx("flex", "flex-1", "items-center", "font-extended", "text-2xl", "overflow-hidden")}>
+        <div className={clsx("py-3", "mr-5")}>
         </div>
         <div className="truncate">{graffiti}</div>
       </div>
-      <div className="font-extended text-2xl ml-2">{pointsStr}</div>
+      <div className={clsx("font-extended", "text-2xl", "ml-2")}>{pointsStr}</div>
     </div>
+       */
```

Comment:
Probably safe to delete this commented code

Reply:
Ah good catch. For whatever reason my editor was rendering that as code not comments so I missed it. Updated.